protected/humhub/modules/user/models/Profile.php

```diff
@@ -413,16 +413,17 @@ public function softDelete()
      *
      * @param string $field
      * @param bool $raw
+     * @param bool $encode
      * @return string|null
      */
-    public function getFieldValue(string $field, bool $raw = false): ?string
+    public function getFieldValue(string $field, bool $raw = false, bool $encode = false): ?string
```

Comment:
@yurabakhtin Wouldn't it be better to also use `true` as the default value here? If all returns were previously encoded?

Reply:
@luke- I find the method is used only in the single place (inside method `User->getDisplayNameSub()`), and there we need `$encode = false` to fix the issue.
We can set the default value to `true` but then we need to change [the code](https://github.com/humhub/humhub/blob/fix/7414-profile-field-encoding/protected/humhub/modules/user/models/User.php#L702) to:

```php
return $this->profile->getFieldValue(Yii::$app->settings->get('displayNameSubFormat', ''), false, false) ?? '';
```

Do you prefer this?

Reply:
@luke- done https://github.com/humhub/humhub/pull/7420/commits/1732d88896268445d7822299283f3d8e10f69b45.